packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-mention.spec.ts

```diff
@@ -588,9 +566,17 @@ describe('RichTextEditorMention', () => {
                 'test:2'
             ]);
         });
+
+        it('user mention button should have `at icon` buttons when user configuration and test elements are given', async () => {
+            await appendUserMentionConfiguration(element);
+            await appendTestMentionConfiguration(element);
+
+            expect(pageObject.getMentionButtonIcon(0)).toBe(ICON_AT_TAG);
+            expect(pageObject.getMentionButtonIcon(1)).toBe(ICON_AT_TAG);
```

Comment:
Let's use a different icon for the test mention configuration in the fixtures file so that these can be distinguished from each other.

Reply:
Replaced with exclamatory icon